Livecheckables/autossh.rb

```diff
@@ -1,6 +1,6 @@
 class Autossh
   livecheck do
     url :homepage
-    regex(/href=.*?autossh-([0-9.]+[a-z]+)\.t/i)
+    regex(/href=.*?autossh[._-]v?(\d+(?:\.\d+)+[a-z]+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?autossh[._-]v?(\d+(?:\.\d+)+[a-z]?)\.t/i)
```

Looking at the [changelog](https://www.harding.motd.ca/autossh/CHANGES.txt), the trailing letter isn't present on the first release (i.e., `1.2` comes before `1.2a`), so we need to make it optional and I think we should only match one character.